docs/06-project-management/readme.md

```diff
@@ -0,0 +1,19 @@
+# Delivery Department
+
+Handbook for CivicActions' delivery department
+
+## Lead
+
+Elizabeth Raley serves as our Director of Professional Services and primary point of contact.
+
+## Index of Contents
+This folder contains the following subpages:
```

Comment:
i feel like these indexes are redundant.  In the readthedocs interface, they are obvious in the left sidebar.  in the github interface, they are visible from the file browser GUI that appears above the readme.  Shouldn't these files just the overview/descriptions?